### DISCUSSION TITLE
S' specific storage

### DISCUSSION THREAD

**Discussion by amyth6** [Deltares/QGIS-Tim#159] [Q&A] [answered]
I am trying to understand the usage of the groundwatermodelling that is avaiable with this Qgis-tim tool. But I still have many things I do not understand. I understand what to do in GIS, but I don't know what the fill in the attribute table for some parameters. So my question is: what is S' specific storage and how can I define this? I found on internet what it means, but I do not understand how you define the specific storage for a certain layer and what S'1/m means. Can someone explain me?

**Comment by MattBrst** [comment]
You could take a look at the [tutorial](https://deltares.github.io/QGIS-Tim/tutorial_TheHague.html) for The Hague to get an idea of what the specific storage could look like. 

If you want to read more about the concept of specific storage, you could use the book Applied Hydrogeology (Fetter). He writes (p. 101 in the 4th edition): _'The storativity S of a confined aquifer is the product of the specific storage (Ss) and the aquifer thickness (b).'_

Since storativity has dimension [-], specific storage has to have dimension [1/L].

**Reply by dbrakenhoff** [reply]
Ah, seeing this now, so you have two answers to choose from ;)

**Comment by dbrakenhoff** [comment, marked as answer]
Hi @amyth6,

The specific storage coefficient is the storage in a (semi-)confined aquifer per meter of aquifer thickness. Or in Wikipedia's more general terms: "Specific storage is the volume of water released from one unit volume of the aquifer under one unit decline in head.". Storativity (S) on the other hand is the amount of storage in the aquifer. The relationship between the two is as follows:

Storativity = specific storage * aquifer thickness

Storativity is a dimensionless parameter, and specific storage has the units 1 / L (e.g. 1/m). The specific storage of the aquifer is almost always unknown and usually some estimate is used, and maybe calibrated e.g. using pumping test data. Values of 1E-3 to 1E-5 are commonly used. My advice would be to plug in some value in that range, and compare your model to observations. 

Hope that helps somewhat!
